### ISSUE TITLE
Focus Appearance (Minimum) move from Operable to Perceivable

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2037]
Working with User Needs and Functional Needs to create User Outcomes, the SCs 2.4.7, 2.4.11 and 2.4.12 end up at Perceivable in my document, not Operable...!

Of course you'll have challenges Operating the UIC if the focus is not clear for you to Perceive it, but that's exactly why it should be under Perceivable in my opinion.

2.4.7 Focus Visible = about visibility (perceiving)
Same for: Contrasting area, Minimum area, Adjacent contrast, and Not fully obscured.

Fact is that even when not "Perceived" well, you can still operate the UICs, thus not beloning to Operable, but Perceivable.

**Comment by jake-abma** [comment]
(also,

see intro text of Operable: **"User interface components and navigation must be operable."**

and Perceivable: **"Information and user interface components must be presentable to users in ways they can perceive."**

)

**Comment by jake-abma** [comment]
+ also there's a very good case where you DO NOT want to activate a control using only keyboard, and Perceiving where the focus is prevent from clicking the wrong UIC. So, operating the UIC is not the problem, but NOT perceiving is.

**Comment by patrickhlauke** [comment]
agree with the sentiment, but short of renumbering, there's likely no clean solution

**Comment by patrickhlauke** [comment]
other than alternatively abandoning the rigid structure of P.O.U.R. and making them more like "tags"/"labels" applied to each SC, so regardless of their number/position in the sequential list, they can be tied to a particular principle (or multiples)

either way though, probably too far gone now / too disruptive

**Comment by jake-abma** [comment]
but 2.4.11 and 2.4.12 are NEW ...

**Comment by patrickhlauke** [comment]
those *could* still be moved, perhaps (though I dread the discussions on that, with the momentum of group decisions)...but then would you leave 2.4.7 where it is (leaving an odd split) or move that too?

sorry, i'm in favour in principle, just not sure if rest of group will just want to go with it / if there's strong appetite to discuss these more disruptive (though correct) decisions at this late stage

**Comment by jake-abma** [comment]
yes, let's know and see issues **in time** before publication (often we discuss only after publication), even still have time to solve, **the moment is right now** to prevent a flaw, but just don't do it... because... Ok, I can live with it... :-)

**Comment by mraccess77** [comment]
2.4.11 and 2.4.12 do seem more appropriate in guideline 1.4.  2.4.7 seems appropriate in principle 2 because without any indicator you can't operate it.

**Comment by alastc** [comment]
From an editorial point of view, it's fine to move now. Might catch a few people out who have already starting integrating it into their docs, but it isn't a recommendation yet...

**Comment by alastc** [comment]
The resolution from the meeting today: Leave focus appearance in operable

We discussed it, it could be in either, but it's association with 2.4.7 won the day.
